content/docs/reference/cli-projects.md

```diff
@@ -79,22 +79,79 @@ In addition to the Neon CLI [global options](/docs/reference/neon-cli#global-opt
 | `--region-id` | The region ID. Possible values: `aws-us-west-2`, `aws-ap-southeast-1`, `aws-eu-central-1`, `aws-us-east-2`, `aws-us-east-1`. Defaults to `aws-us-east-2` if not specified. | string number  |                                       |
 
-#### Example
-
-```bash
-neonctl projects create --name mynewproject --region-id aws-us-west-2
-┌───────────────────┬──────────────┬───────────────┬──────────────────────┐
-│ Id                │ Name         │ Region Id     │ Created At           │
-├───────────────────┼──────────────┼───────────────┼──────────────────────┤
-│ muddy-wood-859533 │ mynewproject │ aws-us-west-2 │ 2023-07-09T17:04:29Z │
-└───────────────────┴──────────────┴───────────────┴──────────────────────┘
-
-┌──────────────────────────────────────────────────────────────────────────────────────┐
-│ Connection Uri                                                                       │
-├──────────────────────────────────────────────────────────────────────────────────────┤
-│ postgres://daniel:<password>@ep-bitter-field-476253.us-west-2.aws.neon.tech/neondb   │
-└──────────────────────────────────────────────────────────────────────────────────────┘
-```
+#### Examples
+
+- Create a project with a user-defined name in a specific region:
+
+    ```bash
+    neonctl projects create --name mynewproject --region-id aws-us-west-2
+    ┌───────────────────┬──────────────┬───────────────┬──────────────────────┐
+    │ Id                │ Name         │ Region Id     │ Created At           │
+    ├───────────────────┼──────────────┼───────────────┼──────────────────────┤
+    │ muddy-wood-859533 │ mynewproject │ aws-us-west-2 │ 2023-07-09T17:04:29Z │
+    └───────────────────┴──────────────┴───────────────┴──────────────────────┘
+
+    ┌──────────────────────────────────────────────────────────────────────────────────────┐
+    │ Connection Uri                                                                       │
+    ├──────────────────────────────────────────────────────────────────────────────────────┤
+    │ postgres://daniel:<password>@ep-bitter-field-476253.us-west-2.aws.neon.tech/neondb   │
+    └──────────────────────────────────────────────────────────────────────────────────────┘
+    ```
+
+    <Admonition type="tip">
+    The Neon CLI provides a `neonctl connection-string` command you can use to extract a connection uri programmatically. See [Neon CLI commands — connection-string](https://neon.tech/docs/reference/cli-connection-string).
+    </Admonition>
+
+- Create a project with the `--output` format of the command set to `json`. This output format returns all of the project response data, whereas the default `table` output format (shown in the preceding example) is limited in the information it can display.
+
+    ```bash
+    neonctl projects create --output json
+    {
+    "0": {
```

Comment:
I've updated the cli so this would return more meaningful indexes, can you please update the example

Reply:
Fixed. Thank you